lib/services/ios-project-service.ts

```diff
@@ -172,7 +172,7 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 			}
 
 			let args: string[] = [];
-			if(this.$options.forDevice) {
+			if(this.$options.forDevice || this.$options.release) {
```

Comment:
You can extract this check to a separate variable and use it here and in the next if :smile_cat: 
But definitely not a merge stopper